dashboard/app/views/layouts/application.html.haml

```diff
@@ -36,6 +36,14 @@
     - if current_user && current_user.age.nil?
       = render partial: 'layouts/age_interstitial'
 
+    #gdpr-dialog
+    - gdpr_data = {}
+    - if current_user
+      - already_accepted = current_user.properties["accepted_data_transfer_agreement"]
+      - gdpr_data[:show_gdpr_dialog] = request.gdpr? && !already_accepted
+      - gdpr_data[:current_user_id] = current_user.id
```

Comment:
I can get the current_user_id from `current_user.id` or from `request.user_id`. I assume they should always be consistent.  Does it matter which one I use?